sass/_animation.scss

```diff
@@ -0,0 +1 @@
+// submit button animation elements will appear here
```

Comment:
Add space as well here at the end of the line.

More on this here => https://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/#:~:text=will%20be%20LF%20.-,core.,to%20denote%20end%20of%20line.

Reply:
fixed